checks/utils/utils.go

```diff
@@ -38,6 +39,7 @@ func GetArguments() Commands {
 	manualInstrumentation := flag.Bool("manual-instrumentation", false, "Provide if your application is using manual instrumentation")
 	debug := flag.Bool("debug", false, "Output debug information")
 	webServer := flag.Bool("web-server", false, "Set if you would like the results served in a web server in addition to console output")
+	enableGrafanaCloudCheck := flag.Bool("enable-grafana-cloud", false, "Enable checking Grafana Cloud credentials")
```

Comment:
Having grafana cloud on the list of components to check should be enough, otherwise you're adding 2 steps to check a single thing. I don't see the advantage or adding another flag on top of that

Reply:
sorry that was a mistake

Reply:
the code was not even used :wink: